pack.yaml

```diff
@@ -6,7 +6,7 @@ keywords:
   - issues
   - ticket management
   - project management
-version: 2.4.2
+version: 2.4.3
```

Comment:
```suggestion
version: 2.5.0
```
also needs an update to match the Changelog

Reply:
done, Thank you!